src/xdocs/config_misc.xml

```diff
@@ -1084,6 +1084,23 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
           times it is practical experience. In any case, this check should
           just ensure that a minimal set of indentation rules is followed.
         </p>
+        <p>
+          Basic offset indentation is used for indentation inside code blocks.
```

Comment:
Can we expand this to all properties so we are clear going forward?

Reply:
I am not sure what to do here, we have properties description for details and examples ...

You are most experienced in this Check, it might be better for you to update as you would like.

Reply:
Done.